src/main/resources/TotemCode/TotemMacros.xml

```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This commit would have been better as a separate commit. This is not a blocker so it can be commited as is but it would be better to avoid this next time.